### ISSUE TITLE
tauri: spurious empty elements in the tab order when navigating in the dialogs

### ISSUE THREAD

**Issue by pvagner** [deltachat/deltachat-desktop#5369]
- Operating System: Linux
- Delta Chat Version: 2.8.0
- Expected behavior: When using tab or shift+tab to navigate in the dialog windows it should only navigate through designed controls.
- Actual behavior: When a dialog is open such as profile or edit profile dialog focus moves into an empty place at the end of a dialog.
- Steps to reproduce the problem:
  * Open delta chat
  * Use tab or shift+tab to navigate to locate the settings button
  * When the profile is displayed, use tab or shift+tab to navigate to locate the edit button
  * Within the edit profile dialog, use tab key to navigate and notice it includes these elements: choose profile picture button, your name entry, bio entry, Cancel button, OK button. When using tauri edition, there are two more elements after the OK button which I am unable to identify.
  * This is only happening with tauri edition, electron edition is not affected.
  * It's happening with other dialogs as well, I have chosen to describe the edit profile dialog as it contains only a few focusable elements usefull for reporting the issue.

**Comment by WofWca** [comment]
Thianks!

This seems to be a WebKit behavior. I just tested it. Those two elements are the document body, then the dialog itself. I'm not sure why WebKit behaves this way.

**Comment by pvagner** [comment]
I have realized I mixed expected and actual behaviour in my original issue description.
So I have corrected it now.
The issue remains, I have just fixed the wording.

**Comment by WofWca** [comment]
You could try it here https://developer.mozilla.org/en-US/play, though the modal dialog is in an `<iframe>` there.
